libraries/Wire/src/Wire.cpp

```diff
@@ -373,7 +373,8 @@ uint8_t TwoWire::requestFrom(uint16_t address, uint8_t size, bool sendStop)
         nonStop = false;
         rxIndex = 0;
         rxLength = 0;
-        err = i2cWriteReadNonStop(num, address, txBuffer, txLength, rxBuffer, size, _timeOutMillis, &rxLength);
+        err = i2cWriteReadNonStop(num, address, txBuffer, txLength, rxBuffer, size, _timeOutMillis, &
```

Comment:
```suggestion
        err = i2cWriteReadNonStop(num, address, txBuffer, txLength, rxBuffer, size, _timeOutMillis, &rxLength
```

Reply:
Oups, most likely deleted that by accident when doing searches